tests/denops/runtime/functions/denops/request_async_test.ts

```diff
@@ -74,43 +74,86 @@ testHost({
       // Load plugin and wait.
       await host.call("execute", [
         "let g:__test_denops_events = []",
-        `call denops#plugin#load('dummyLoaded', '${scriptValid}')`,
+        `call denops#plugin#load('dummy', '${scriptDispatcher}')`,
       ], "");
       await wait(async () =>
         (await host.call("eval", "g:__test_denops_events") as string[])
-          .includes("DenopsPluginPost:dummyLoaded")
+          .includes("DenopsPluginPost:dummy")
       );
-      await host.call("execute", [
-        "let g:__test_denops_events = []",
-      ], "");
 
-      outputs = [];
-      await host.call(
-        "denops#request_async",
-        "dummyLoaded",
-        "test",
-        ["foo"],
-        "TestDenopsRequestAsyncSuccess",
-        "TestDenopsRequestAsyncFailure",
-      );
+      await t.step("returns immediately", async () => {
+        await host.call("execute", [
+          "let g:__test_denops_events = []",
+        ], "");
+
+        await host.call(
+          "denops#request_async",
+          "dummy",
+          "test",
+          ["foo"],
+          "TestDenopsRequestAsyncSuccess",
+          "TestDenopsRequestAsyncFailure",
+        );
 
-      await t.step("returns immediately", () => {
-        assertEquals(outputs, []);
+        assertEquals(await host.call("eval", "g:__test_denops_events"), []);
+      });
+
+      await t.step("calls dispatcher method", async () => {
+        await delay(100 + ASYNC_DELAY);
```

Comment:
Maybe `wait` instead of `delay`?

Reply:
Since the test plugin defines `delay(100)`, we use delay accordingly.